### ISSUE TITLE
[Adaptive Card iOS - Flight Update]: Voice Over is focus is landing on the hidden place while navigating using the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#26]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**

1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'FlightUpdate.JSON' and double tap to activate it.
7. The Card will open.
8. Perform right swipe, navigate to AMS.
9. Again perform right swipe and observe the issue.

**Actual Result:**
While navigating the 'Flight Update' Card using swipe gesture, when focus is on AMS then on performing right swipe, voice over focus islanding on the hidden place and announcing as Space, group. 

Expected Result:
Voice Over focus should not land on the hidden place and when the focus is on AMS then on performing right swipe it should be lands on the 'Flight to' image.

**Note:**
This issue is not repro using keyboard.

**User Impact:**
It will impact the screen reader user as it will take extra effort to navigate on the card which will consume extra time for the screen reader user.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

 

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/1f8e12fd-3405-419e-8167-253d9df26404

**Comment by GabrielMedAlv** [comment]
Not reproducible on latest version. Working as expected using swipe gesture to move. Attached evidence.

https://github.com/microsoft/AdaptiveCards-Mobile/assets/130393016/caa3ab34-d9d1-40a3-8324-0059310fde30

**Comment by vagpt** [comment]
This issue is still repro on the build 1.0 (2.4.0-beta.24.11.3.1), hence reactivating this issue.

**Comment by GabrielMedAlv** [comment]
Merged PR with fix.
